### PR TITLE
[DOC] 0.7.1 release note

### PR DESCRIPTION
**Pull request by minahlee**
### What is this PR for?
Add 0.7.1 release note

### What type of PR is it?
Documentation

### Todos
* [x] Confirm release date and update list of people who verified release once vote pass.

### Screenshots (if appropriate)
**Download page**
screencapture-localhost-4000-download-html-1490930121638
**Release note**
screencapture-localhost-4000-releases-zeppelin-release-0-7-1-html-1490930133953

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: releases/zeppelin-release-0.7.1.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+The community put significant effort into improving Apache Zeppelin since the last release focusing on restarting interpreter process stability, python interpreter improvement, table/chart rendering bug fix.
+24 contributors provided 80+ patches for improvements and bug fixes.
+More than 70+ issues have been resolved.
+
+We encourage to [download](../../download.html) the latest release. Feedback through the [mailing lists](../../community.html) is very welcome.
+
+### Improvements
+   * Rewrite python interpreter for streaming output and seamless matplotlib support
+   * Make dynamic select form turn on or off using checkbox
```
Comment (AhyoungRyu) [inline releases/zeppelin-release-0.7.1.md]: After #2156, we can do same with checkbox dynamic form :)

Comment (felixcheung) [inline releases/zeppelin-release-0.7.1.md]: interesting - I tested with rc3, switch to report mode, but didn't catch any popup or prompt to let me know how to turn this off? I'm not sure - should I see that?

File: supported_interpreters.md [hunk 2]
```diff
@@ -95,13 +99,26 @@ Please check the below table before you download Zeppelin package.
       <a href="http://tajo.apache.org/" target="_blank">Tajo</a>
       are available
     </td>
+    <td>
+      <a href="http://www.postgresql.org/" target="_blank">PostgreSQL</a>,
+      <a href="https://www.mysql.com/" target="_blank">MySQL</a>,
+      <a href="https://mariadb.org/" target="_blank">MariaDB</a>,
+      <a href="https://aws.amazon.com/documentation/redshift/" target="_blank">Redshift</a>,
+      <br/>
```
Comment (felixcheung) [inline supported_interpreters.md]: why is there a `<br/>` here?

File: releases/zeppelin-release-0.7.1.md (hunk 1 above)
Comment (AhyoungRyu) [inline releases/zeppelin-release-0.7.1.md]: @felixcheung You mean it's not intuitive, right? Currently there is no way to let ppl know they can turn it on/off. Let me improve it in the near future :)

Comment (felixcheung) [inline releases/zeppelin-release-0.7.1.md]: right thanks! I was actually just wondering what to expect :) I remember your PR on getting a tooltip/popup but I think that doesn't show in report mode?

Comment (AhyoungRyu) [inline releases/zeppelin-release-0.7.1.md]: Yeah probably this one #2072. The tooltip was shown in report mode as well. But at that time, I wasn't sure that showing tooltip is good or not. Maybe we can combine the idea in #2072 and #2100 for the improvement.

File: supported_interpreters.md (hunk 2 above)
Comment (minahlee) [inline supported_interpreters.md]: I just copy and paste it from [other line](https://github.com/apache/zeppelin/pull/2209/files/3c836008ba81dc1216da3c7bc259ea9f92c14ff3#diff-b4f569e0e53323a3f20dc62535bfac06R95). I guess its for readability. Left cell is without `<br/>` right cell is with `<br/>`
screen shot 2017-03-31 at 5 28 54 pm

File: releases/zeppelin-release-0.7.1.md (hunk 1 above)
Comment (minahlee) [inline releases/zeppelin-release-0.7.1.md]: @AhyoungRyu updated

Comment (AhyoungRyu) [inline releases/zeppelin-release-0.7.1.md]: @minahlee Thanks!

**Comment by minahlee** [comment]
Merge if there is no more discussion for new release publish

**Comment by Leemoonsoo** [comment]
LGTM

**Pull request closed without merging**